torch/distributions/gumbel.py

```diff
@@ -1,6 +1,7 @@
 from numbers import Number
 
 import math
+from numpy import euler_gamma
```

Comment:
This introduces a hard depenendcy on NumPy. Is it desired?

Reply:
No we should simply hard code this number:
```py
euler_gamma =  0.57721566490153286060
```

Reply:
Ah, yes. You are right about the hard dependency. But, I suppose NumPy is required for PyTorch. @apaszke could you suggest what should be done?